src/main/cluster-manager.ts

```diff
@@ -1,24 +1,21 @@
 import "../common/cluster-ipc";
 import type http from "http";
 import { ipcMain } from "electron";
-import { action, autorun, observable, reaction, toJS } from "mobx";
+import { action, autorun, reaction, toJS } from "mobx";
 import { ClusterStore, getClusterIdFromHost } from "../common/cluster-store";
 import { Cluster } from "./cluster";
 import logger from "./logger";
 import { apiKubePrefix } from "../common/vars";
 import { Singleton } from "../common/utils";
-import { CatalogEntity, catalogEntityRegistry } from "../common/catalog";
+import { catalogEntityRegistry } from "../common/catalog";
 import { KubernetesCluster } from "../common/catalog-entities/kubernetes-cluster";
 
 const clusterOwnerRef = "ClusterManager";
 
 export class ClusterManager extends Singleton {
-  catalogSource = observable.array<CatalogEntity>([]);
-
   constructor(public readonly port: number) {
     super();
 
-    catalogEntityRegistry.addObservableSource("lens:kubernetes-clusters", this.catalogSource);
```

Comment:
Was probably forgotten during the original PR. We don't need this now that the source is `KubeconfigSync`.

Reply:
See #2663, there is a lot that I knew I didn't to do. Was planning to do this in the future.